core/controllers/community_dashboard.py

```diff
@@ -78,6 +83,109 @@ def get(self, opportunity_type):
 
         self.render_json(self.values)
 
+    def _get_skill_opportunities_with_topic(self, cursor):
```

Comment:
`...with_corresponding_topic_name` or something around?

with the current name:
- I'm not sure how skills and topics are connected. 
- I'm assuming it returns the entire topic details as well.